### DISCUSSION TITLE
Generating standalone Package

### DISCUSSION THREAD

**Discussion by CarlPhilippHohl** [carla-simulator/carla#6774] [Q&A]
Hi,

I am trying to generate a standalone package with "make package ARGS="--packages=MyPackage". However, the makefile returns
"unkown option --packages". If i remove the ARGS part, it just builds everything.
Anyone knows a fix for this one? What is the correct usage of this command?

Best Regards

**Comment by maybe2122** [comment]
I encountered this issue as well, I dont know where to put my package and  which pakcage can be packaged
